### ISSUE TITLE
zenbot backfill start and end date bug

### ISSUE THREAD

**Issue by hanassabio** [DeviaVir/zenbot#909]
![image](https://user-images.githubusercontent.com/34634608/34085420-9409b142-e388-11e7-9d21-391a272ec382.png)

 **I am trying to use the syntax for constricted time range of backfilling, yet it is not executing. Please advise!**

![image](https://user-images.githubusercontent.com/34634608/34085417-8a469274-e388-11e7-8677-1cb3f9804faa.png)

**Comment by DeviaVir** [comment]
It looks like backfill is just `--days`, train does have `--start` and `--end`, so looks like a documentation problem.

**Comment by hanassabio** [comment]
Thanks @DeviaVir for clarifying. Do you know if there is any other way to go about it to just consider a certain time frame instead of previous days?

**Comment by hanassabio** [comment]
For the <"timestamp"> when running the zenbot, use the Unix syntax timestamp in milliseconds, **not** as advised in the documentation (YYYYMMDDhhmm). It then works as expected. 

Free converter here: https://www.epochconverter.com/

**Comment by acecilia** [comment]
Hello @hanassabio:
I tried `backfill --start=1496620800000 --end=1499212800000` and also `backfill --start="1496620800000" --end="1499212800000"`
In both cases I keep getting:

    error: unknown option `--start'

Any ideas? Thanks!

**Comment by hanassabio** [comment]
Hi, are you including the "selector" in your code after _backfill_ and before _--start_?  See below taken from the main documentation:
![image](https://user-images.githubusercontent.com/34634608/34329259-de8e6e3e-e8f0-11e7-96e0-788fa22721b6.png)

**Comment by acecilia** [comment]
@hanassabio I think I am doing it right:

![captura de pantalla 2017-12-25 a las 14 05 13](https://user-images.githubusercontent.com/7938671/34333957-b5dd9348-e97c-11e7-9394-5d41e2a92f60.png)

I am looking at the source code for backfill and I do not see any --start or --end option around

**Comment by ghost** [comment]
@acecilia  found today your issue, maybe late, but the solution is to remove the to underscore in front of 'start' and 'end' --> wrong --start & --end  -> Solution start & end . Simple, it work find for me. I hope it will work for you.

**Comment by DJCrashdummy** [comment]
neither of the suggested solutions work. :frowning_face: 

and at the readme there is still the completely wrong syntax mentioned: https://github.com/DeviaVir/zenbot#run-a-simulation-for-your-selector
so this issue is definitely **not solved and needs to be reopened**!

whereby the best would be to enable backfilling between certain dates (again) - no matter with which syntax, because if i just want to test a specific month/week/etc and have to backfill all data between now and then, it is just a waste of time, space, bandwidth and whatever resources else...

**Comment by durek1337** [comment]
The problem still exists!

The command
> zenbot backfill binance.ZEN-BTC start="1553263560000" end="1553904000000"

results into the same as

> zenbot backfill binance.ZEN-BTC

Yes, I also tried the format "YYYYMMDDhhmm".
It always loads the trades of the last 14 days and never the span which would be very helpful.

edit: I forked the repo and I will try to add this feature. I don't know how long it would take but there is a branch for the mode 'backward' with 'target_time' and else 'start_time' and 'target_time'. So maybe the functionality still exists rawly? If someone who is more familiar with the code wants to do the job, i would highly appreciate it.

edit2: I guess I fixed it. Really really simple thing. It's the first time for me participating on open source so after testing I have to look for the conventional process to add it. Stay tuned, I will do this the next days.

edit3: pull request done.

> zenbot backfill \<selector\> --start="\<unixtime in ms\>" --end="\<unixtime in ms\>"

will work after the merge
